extension/observer/k8sobserver/README.md

```diff
@@ -68,14 +68,15 @@ This spec-determined value would then be available via the `${env:K8S_NODE_NAME}
 
 All fields are optional.
 
-| Name | Type | Default | Docs |
-| ---- | ---- | ------- | ---- |
-| auth_type | string | `serviceAccount` | How to authenticate to the K8s API server.  This can be one of `none` (for no auth), `serviceAccount` (to use the standard service account token provided to the agent pod), or `kubeConfig` to use credentials from `~/.kube/config`. |
-| node | string | <no value> | The node name to limit the discovery of pod, port, and node endpoints. Providing no value (the default) results in discovering endpoints for all available nodes. |
-| observe_pods | bool | `true` | Whether to report observer pod and port endpoints. If `true` and `node` is specified it will only discover pod and port endpoints whose `spec.nodeName` matches the provided node name. If `true` and `node` isn't specified, it will discover all available pod and port endpoints. Please note that Collector connectivity to pods from other nodes is dependent on your cluster configuration and isn't guaranteed. | 
-| observe_nodes | bool | `false` | Whether to report observer k8s.node endpoints. If `true` and `node` is specified it will only discover node endpoints whose `metadata.name` matches the provided node name. If `true` and `node` isn't specified, it will discover all available node endpoints. Please note that Collector connectivity to nodes is dependent on your cluster configuration and isn't guaranteed.| 
-| observe_services | bool | `false` | Whether to report observer k8s.service endpoints.|
-| observe_ingresses | bool | `false` | Whether to report observer k8s.ingress endpoints.|  
+| Name              | Type | Default | Docs |
+|-------------------|  | ------- | ---- |
```

Comment:
I was checking out the readme today and found that the table isn't rendering properly because there's a formatting issue in the table header row.
I guess this is a typo, can we please fix it?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40190